sigh/lib/sigh/runner.rb

```diff
@@ -65,7 +65,7 @@ def profile_type
     # Fetches a profile matching the user's search requirements
     def fetch_profiles
       UI.message "Fetching profiles..."
-      results = profile_type.find_by_bundle_id(Sigh.config[:app_identifier]).find_all(&:valid?)
+      results = profile_type.find_by_bundle_id(Sigh.config[:app_identifier])
```

Comment:
Oh, right, I didn't think about this.

I'm not yet sure about this change, as it changes the way sigh works for the different use-cases. I'll give it some more thought.